### ISSUE TITLE
[macOS] Regression: Several games fail to boot after PR 837 (Bindings)

### ISSUE THREAD

**Issue by shinra-electric** [shadps4-emu/shadPS4#869]
At least the following games no longer boot on macOS after PR #837:

Dead Cells - CUSA11253
Limbo - CUSA01369

They fail with
```
[Debug] <Critical> liverpool.h:unhandled_exception:1110: Unreachable code!
Unhandled exception: vk::Device::createComputePipelineUnique: ErrorInvalidShaderNV
```

**Comment by squidbus** [comment]
For future reference, Vulkan Validation enabled in Dead Cells yields this:
```
[Render.Vulkan] <Error> vk_platform.cpp:DebugUtilsCallback:71: VUID-VkShaderModuleCreateInfo-pCode-08737: Validation Error: [ VUID-VkShaderModuleCreateInfo-pCode-08737 ] | MessageID = 0xa5625282 | vkCreateShaderModule(): pCreateInfo->pCode (spirv-val produced an error):
Expected Image 'Sampled Type' to be the same as Texel components
  OpImageWrite %91 %92 %93
. The Vulkan spec states: If pCode is a pointer to SPIR-V code, pCode must adhere to the validation rules described by the Validation Rules within a Module section of the SPIR-V Environment appendix (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkShaderModuleCreateInfo-pCode-08737)
```
Validation error is not Mac specific but does cause a crash there.

**Comment by shinra-electric** [comment]
Fixed by #871
